cmd/scollector/collectors/program.go

```diff
@@ -151,12 +178,20 @@ Loop:
 			Value:     val,
 			Tags:      opentsdb.TagSet{"host": util.Hostname},
 		}
-		for _, tag := range sp[3:] {
-			tags, err := opentsdb.ParseTags(tag)
+		for _, field := range sp[3:] {
+			if strings.HasPrefix(field, "meta.") {
```

Comment:
@mjibson How do you identify which metric + tagset the metadata belongs to without repeating most of the information on the original line anyways?